### PR TITLE
Default to npipe address on Windows

### PR DESCRIPTION
**Pull request by aanand**
See https://github.com/docker/compose/issues/3701.

ping @dgageot

### CODE REVIEW

**Comment by shin-** [comment]
Do you think this is too early? The `npipe` code hasn't really been tested extensively yet. I'd wait for 1.10

File: docker/utils/utils.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def parse_host(addr, is_win32=False, tls=False):
     path = ''
 
     if not addr and is_win32:
-        addr = '{0}:{1}'.format(DEFAULT_HTTP_HOST, 2375)
+        addr = 'npipe:////./pipe/docker_engine'
```
Comment (shin-) [inline docker/utils/utils.py]: Maybe this should be a constant

**Comment by aanand** [comment]
Makes sense to me. I'll move it to the 1.10 milestone.

**Comment by shin-** [comment]
Cool, LGTM :+1:

**Pull request merged**